src/.clang-format

```diff
@@ -82,8 +103,32 @@ PenaltyBreakString: 1000
 PenaltyExcessCharacter: 1000000
 PenaltyReturnTypeOnItsOwnLine: 60
 
+# includes
+IncludeCategories:
```

Comment:
I am not familiar with this but I am presuming this is a different way to try and position the pre/post\_guard.h files to avoid the hard-wired "turn clang-format off" around them that I previously tried. :+1:  if it works.